source/language/subroutines.rst

```diff
@@ -14,9 +14,16 @@ the subroutine by reference or name, while classical types are passed in by valu
 All arguments are declared together with their type. For example, ``qubit ancilla``
 defines a quantum bit argument named ``ancilla``.
 
-The subroutines return up to one value of classical type, signified by the
-``return`` keyword. If there is no return type, the empty ``return``
-keyword may be used to immediately exit from the subroutine.
+Subroutines return up to one value of classical type, signified by the
+``return`` keyword. If there is no return value, the empty ``return``
+keyword may be used to immediately exit from the subroutine. Subroutines that
+do not return a value may be declared with no output signifiers::
+
+    def name(parameters) { body }
+
+or may be declared to return the :ref:`void type <void-type>`::
+
+    def name(parameters) -> void { body }
```

Comment:
It might be worth adding an extra sentence after this just to clarify that "no specified return" and `-> void` are exactly equivalent (assuming that they are).

Reply:
Added in https://github.com/openqasm/openqasm/pull/384/commits/73136b3c45e5e6381e771d93c071e8758761a19e